### DISCUSSION TITLE
MongoDB as a database for Glowroot central !!

### DISCUSSION THREAD

**Discussion by iam8139** [glowroot/glowroot#819] [Ideas]
Is there any support to use Mongo DB as a database instead of Cassandra??

**Comment by iam8139** [comment]
Mongo DB also supports newer Java versions...

**Comment by lanmaster** [comment]
May be RRD4J will be more useful? It process upon limited size files...
No more DB fragmentation...

**Comment by Never-lab** [comment]
**No** — Glowroot Central only supports **Cassandra** today (plus **embedded H2** in all-in-one agent mode). There is no MongoDB backend or configuration switch.

If the question is “can I use this now?” → answer is no. If you want a different storage engine, that’s a feature proposal (Ideas is fine). For “is this supported?” threads, **Q&A** is usually a better category (#1223).
